serverless/aws/lambda/src/main/scalajs/sttp/tapir/serverless/aws/lambda/js/package.scala

```diff
@@ -0,0 +1,14 @@
+package sttp.tapir.serverless.aws.lambda
+
+import scala.concurrent.{ExecutionContext, Future}
+
+package object js {
+  type JsRoute[F[_]] = AwsJsRequest => F[AwsJsResponse]
+
+  implicit class ToJsRouteConverter(route: Route[Future]) {
+    def toJsRoute(implicit ec: ExecutionContext): JsRoute[Future] = {
```

Comment:
do we need the EC here? shouldn't this work for any `F[_]`?

Reply:
Moved to `AwsJsRouteHandler` and made private